libs/blocks/global-footer/global-footer.js

```diff
@@ -72,12 +72,7 @@ class Footer {
     this.body = await fetchAndProcessPlainHtml({
       url,
       shouldDecorateLinks: false,
-    })
-      .catch((e) => lanaLog({
-        message: `Error fetching footer content ${url}`,
-        e,
-        tags: 'errorType=error,module=global-footer',
-      }));
+    });
```

Comment:
What happens if the `fetchAndProcessPlainHtml` has a JS error - is the catch not needed?

Reply:
if there is any JS error in `fetchAndProcessPlainHtml` it will get logged with `logErrorFor` used with `decorateContent` in this case. Similar way it will get logged at other places as well.